.gitignore

```diff
@@ -3,3 +3,5 @@
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+.idea/*
```

Comment:
What's this for?

Reply:
This is for my PyCharm IDE, I added it so I wouldn't see it in my Git output.